appservice/src/intTest/groovy/nu/yona/server/AbstractAppServiceIntegrationTest.groovy

```diff
@@ -148,9 +148,13 @@ abstract class AbstractAppServiceIntegrationTest extends Specification
 		analysisService.postToAnalysisEngine(user, categories, url, YonaServer.relativeDateTimeStringToZonedDateTime(relativeDateTimeString))
 	}
 
-	void assertWeekOverviewBasics(response, numberOfReportedGoals)
+	void assertWeekOverviewBasics(response, numberOfReportedGoals, expectedTotalElements)
 	{
+		def expectedPageSize = 2
```

Comment:
The next method makes this a default parameter. Can't you apply that elegant solution here too?